### ISSUE TITLE
Local error graph only updates for original expression not alternates.

### ISSUE THREAD

**Issue by zaneenders** [herbie-fp/odyssey#189]
Herbie: 9287265 Odyssey: https://github.com/herbie-fp/odyssey/commit/8da44443c78303d006448a7558c8c626f2c2104f.

https://github.com/user-attachments/assets/e1e1decf-151a-4348-85fa-c948b7248de6

**Comment by zaneenders** [comment]
Ok this isn't always true.


https://github.com/user-attachments/assets/02476cc3-e9bf-40df-a48a-99a4c8ebfdbb

**Comment by elmisback** [comment]
I don't think we expect to see anything changing in .5/n with that amount of error? It would be like 1-7% red, which might not be visible. I can check it out though

Please copy the spec being tested so we can reproduce issues like these!
